design-proposals/generated-disk-serial.md

```diff
@@ -0,0 +1,120 @@
+# Overview
+This proposal introduces a new feature in KubeVirt to automatically
+generate serial numbers for each disk attached to a virtual machine.
+
+## Motivation
+Some applications running in the VM require the disk serial number to be defined and persistent across reboots.
+Generating the serial reduces manual configuration, and in the future it can be part of a `VirtualMachinePreference` for the OS. 
```

Comment:
nit: How it's configured manually today by the user? What if the user already configured something and you will generate a new serial number?

Reply:
The serial number can be set manually by the user in the path that I've mentioned in the above comment.

If a serial number is already specified, it will not be generated. I will mention it in the document.